src/fw/applib/ui/kino/kino_layer.c

```diff
@@ -35,7 +72,17 @@ static void prv_update_proc(Layer *layer, GContext *ctx) {
   }
 
   const GRect reel_bounds = kino_layer_get_reel_bounds(kino_layer);
-  kino_player_draw(&kino_layer->player, ctx, reel_bounds.origin);
+  // Check the inversion flag stored in the context pointer's LSB
+  if (KINO_LAYER_GET_INVERT_FLAG(kino_layer)) {
+    GDrawCommandProcessor gdraw_inv_processor = {
+        .command = prv_invert_pdc_colors,
+    };
+
+    KinoReelProcessor invert_processor = {.draw_command_processor = &gdraw_inv_processor};
```

Comment:
This will similarly require values for `bitmap_processor`. Of course, this could happen at a follow-up PR. My recommendation would be to not solve with via `GBitmapProcessor.pre` or `.post` but a new field `composite_mode` for a lot of orthogonal power.